src/script/components/calling/GroupVideoGridTile.tsx

```diff
@@ -43,6 +43,24 @@ export interface GroupVideoGridTileProps {
   selfParticipant: Participant;
 }
 
+const getParticipantNameColor = ({
+  isActivelySpeaking,
+  isAudioEstablished,
+}: {
+  isActivelySpeaking: boolean;
+  isAudioEstablished: boolean;
+}) => {
+  if (!isAudioEstablished) {
+    return 'var(--gray-60)';
```

Comment:
if anybody's wondering, the name label has the same bg color in light and dark mode, so a static color is fine here :+1: